doc/en/fcrontab.1.sgml

```diff
@@ -53,22 +53,26 @@ A copy of the license is included in gfdl.sgml.
 	    linkend="fcron.8">&fcron;(8)</link> daemon. As &fcron; internally
 	    uses a non-human readable format (this is needed because &fcron;
 	    saves more informations than the user gives, for example the time
-	    and date of next execution), the user cannot edit directly his
-	    &fcrontabf; (the one used by &fcron;).</para> <para>When a user
+	    and date of next execution), the user cannot edit his &fcrontabf;
+		directly (the one used by &fcron;).</para> <para>When a user
 	    installs a &fcrontabf;, the source file is saved in the spool
-	    directory (<filename>&fcrontabsdir;</filename>) to allow future
-	    editions, and a formatted file is generated for the &fcron; daemon,
-	    which is signaled once about ten seconds before the next minute for
-	    all changes made previously. The daemon is not informed of the changes
+	    directory (<filename>&fcrontabsdir;</filename>) to allow it to be
+	    edited, and a formatted file is generated for the &fcron; daemon.
+		This formatted (non human readable) format is sent to the daemon
+		once per minute (about ten seconds before the full minute) with all
```

Comment:
this is now configurable at compile time, with ./configure's `--with-max-fcrontab-reload-delay-seconds`. Do you mind rephrasing as sent with a delay, which by default is once per minute [...] ?

Reply:
yes, added slight re-wording around this